File: app/app/src/main/java/com/nldining/app/models/Restaurant.kt
package com.nldining.app.models

import java.io.Serializable

data class Restaurant(
    val id: Int,
    val naam: String,
    val categorie: String,
    val tags: String,
    val adres: String,
    val reviewScoreOverall: Double,
    val reviewScoreFood: Double,
    val reviewScoreService: Double,
    val reviewCount: Int,
    val lastReview: String,

    val latitude: Double? = null,
    val longitude: Double? = null
) : Serializable

File: NLDiningApi/NLDiningApi/Models/Restaurant.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Table("restaurants")]
public class Restaurant
{
    [Key]
    [Column("Id")]
    public int Id { get; set; }

    [Column("Name")]
    public string Naam { get; set; }

    [Column("Category")]
    public string Categorie { get; set; }

    [Column("Tags")]
    public string Tags { get; set; }

    [Column("Address")]
    public string Adres { get; set; }

    [Column("ReviewScoreOverall", TypeName = "decimal(4,2)")]
    public decimal? ReviewScoreOverall { get; set; }

    [Column("ReviewScoreFood", TypeName = "decimal(4,2)")]
    public decimal? ReviewScoreFood { get; set; }

    [Column("ReviewScoreService", TypeName = "decimal(4,2)")]
    public decimal? ReviewScoreService { get; set; }

    [Column("ReviewScoreAmbiance", TypeName = "decimal(4,2)")]
    public decimal? ReviewScoreAmbiance { get; set; }

    [Column("ReviewCount")]
    public int ReviewCount { get; set; }

    [Column("LastReview")]
    public string LastReview { get; set; }
}